stats.rst

```diff
@@ -0,0 +1,308 @@
+.. _stats:
+
+=================
+Stats Quick Start
+=================
+
+A quick guide to getting the internal stats collection working in flux.
+This guide assumes you have already built a flux binary; if you have not, see :ref:`quickstart` to get started.
```

Comment:
You can probably just delete these two statements (the first isn't a sentence).

I'd suggest replacing with a sentence that describes what "stats" is, e.g.

> Flux includes an optional instrumentation facility to collect internal metrics, or "stats", and send them to an aggregator for eventual display in a backend graphing tool such as Graphite.